index.html

```diff
@@ -101,7 +101,7 @@ <h2>Project name goes here</h2>
             </div>
             
           <div class="project-container project-3">
-            <img src="images/project-3.jpg" alt="project-3 image">
+            <img src="images/project-3.jpg" alt="macbook pro 16 frontal view and a smartphone">
 
             <div class="single-project-info">
               <h2>Project name goes here</h2>
```

Comment:
- [ ] Please change this to `h3` instead of `h2` as the Heading levels should have a meaningful hierarchy.